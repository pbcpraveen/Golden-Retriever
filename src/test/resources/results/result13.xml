<?xml version="1.0" encoding="UTF-8" standalone="no"?><result><result><PERSONA>CINNA, a poet. </PERSONA><PERSONA>Senators, Citizens, Guards, Attendants, &amp;c.</PERSONA><PERSONA>CALPURNIA, wife to Caesar.</PERSONA><PERSONA>A Soothsayer</PERSONA><PERSONA>ARTEMIDORUS Of Cnidos, a teacher of rhetoric. </PERSONA><PERSONA>PINDARUS, servant to Cassius.</PERSONA><PERSONA>PORTIA, wife to Brutus.</PERSONA></result></result>