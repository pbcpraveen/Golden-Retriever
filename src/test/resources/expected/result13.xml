<?xml version="1.0" encoding="UTF-8" standalone="no"?><result><result><PERSONA>CALPURNIA, wife to Caesar.</PERSONA><PERSONA>ARTEMIDORUS Of Cnidos, a teacher of rhetoric. </PERSONA><PERSONA>A Soothsayer</PERSONA><PERSONA>CINNA, a poet. </PERSONA><PERSONA>PORTIA, wife to Brutus.</PERSONA><PERSONA>PINDARUS, servant to Cassius.</PERSONA><PERSONA>Senators, Citizens, Guards, Attendants, &amp;c.</PERSONA></result></result>